src/Test.UnitTests.Sarif/Test.UnitTests.Sarif.csproj

```diff
@@ -8,7 +8,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
>net461 [](start = 36, length = 6)

Failure to remove this target was an oversight in a previous build clean-up change. Noticed it only in the VS Test Explorer UI.